src/toolkits/feature_engineering/word_counter.cpp

```diff
@@ -125,7 +129,10 @@ flexible_type word_counter_apply_with_manual(const flexible_type& input,
         if (kvp.second.get_type() != flex_type_enum::INTEGER
             && kvp.second.get_type() != flex_type_enum::FLOAT)
           log_and_throw("Invalid type. Dictionary input to WordCounter must have integer or float values.");
-        word_count_delimiters_update(kvp.first.get<flex_string>(), delimiter_list, to_lower, ret_count);
+	if (kvp.second.get_type() == flex_type_enum::INTEGER)
+            word_count_delimiters_update(kvp.first.get<flex_string>(), delimiter_list, to_lower, ret_count, run_mode, kvp.second.get<flex_int>());
```

Comment:
formatting for function calls - https://google.github.io/styleguide/cppguide.html#Function_Calls